src/main/java/core/basesyntax/figures/Square.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.figures;
+
+import java.util.Locale;
+
+public class Square extends Figure {
+    private double side;
+    private double area;
+
+    public Square(String color, double side) {
+        super(color);
+        this.side = side;
+        this.area = calculateArea();
+    }
+
+    public double getSide() {
+        return side;
+    }
+
+    public void setSide(double side) {
+        this.side = side;
+        this.area = calculateArea();
+    }
+
+    public double getArea() {
+        return area;
+    }
```

Comment:
Instead of returning a stored value, 'getArea()' should calculate the area at the time it is called to ensure the value is always accurate.